README.md

```diff
@@ -83,6 +83,17 @@ If using Rails, this is evaluated against the timezone configured in Rails, othe
 If you want to have your jobs enqueued based on a different time zone you can specify a timezone in the cronline,
 like this `'0 22 * * 1-5 America/Chicago'`.
 
+#### Natural-language formats
+
+Since `v1.3.0`, you can use the natural-language formats supported by Fugit, such as:
```

Comment:
```suggestion
Since sidekiq-cron `v1.3.0`, you can use the natural-language formats supported by Fugit, such as:
```

Reply:
```suggestion
Since sidekiq-cron `v1.3.0`, you can use the natural-language formats supported by Fugit, such as:
```

Reply:
hey @Spone maybe I'm missing something, what this will be available from next release, which is going to be `v1.7.0` (current one, released yesterday, was `v1.6.0`).

Reply:
I thought so but here is what the changelog says: https://github.com/ondrejbartas/sidekiq-cron/blob/master/CHANGELOG.md?plain=1#L33

Reply:
OK I fixed it!